src/utils.jl

```diff
@@ -553,8 +553,9 @@ zlims(sp_idx::Int = 1) = zlims(current(), sp_idx)
 
 function get_clims(sp::Subplot)
     zmin, zmax = Inf, -Inf
+    z_colored_series = (:contour, :contour3d, :heatmap, :histogram2d, :surface)
```

Comment:
Could you just use `like_surface(series[:seriestype])`?

Reply:
`like_surface` gives true for `wireframe` and `image` which aren't colored by z so I don't think so.